evaluations/yieldscan_1_mmagician.md

```diff
@@ -0,0 +1,44 @@
+|--------|----------------------------|------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                    | <ul><li>[x] </li></ul> | <ul><li>[yieldscan-frontend](https://github.com/yieldscan/yieldscan-frontend/blob/master/LICENSE.md)</li><li>[yieldscan-backend-crawler](https://github.com/yieldscan/yieldscan-backend-crawler/blob/master/LICENSE.md)</li><li>[yieldscan-backend-api](https://github.com/yieldscan/yieldscan-backend-api/blob/master/LICENSE.md)</li></ul>                                         |                                                                                                                                                               |
+|    0b. | Documentation              | <ul><li>[ ] </li></ul> | <ul><li>[yieldscan-frontend documentation](https://github.com/yieldscan/yieldscan-frontend/tree/develop#codebase-overview)</li><li>[yieldscan-backend-crawler documentation](https://github.com/yieldscan/yieldscan-backend-crawler#codebase-overview)</li><li>[yieldscan-backend-api documentation](https://github.com/yieldscan/yieldscan-backend-api#codebase-overview)</li></ul> | Documentation regarding the key part of each repo could be expanded, e.g. for the crawler repo, describing in more detail what the crawlers are, what they do |
+|    0c. | Testing Guide              | <ul><li>[ ] </li></ul> | https://github.com/yieldscan/yieldscan-frontend/tree/develop#tests                                                                                                                                                                                                                                                                                                                   | Some helper commands that check the state of the db/sync would come in handy                                                                                  |
```

Comment:
have pushed changes for this too, now one can simply check the db sync via `/api/<network>/historic/last-era-index` and also added more logs for crawler regarding the db sync